src/Cake.Svn/SvnAliases.Delete.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+using Cake.Svn.Delete;
+
+namespace Cake.Svn
+{
+    public static partial class SvnAliases
+    {
+        /// <summary>
+        /// Delete a directory from Subversion.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directory">The directory to delete.</param>
+        /// <returns>
+        /// <c>true</c> if the command was successful. Otherwise <c>false</c> will be returned.
+        /// </returns>
+        /// <example>
+        /// <para>Delete a directory from Subversion.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var directoryRemoved = SvnDelete(@"C:\project\src\newfolder\");
+        ///
+        ///     Verbose("Directory removed: {0}", directoryRemoved);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Delete")]
+        [CakeNamespaceImport("Cake.Svn.Delete")]
+        public static bool SvnDelete(this ICakeContext context, DirectoryPath directory)
```

Comment:
Will be ambiguous between  `SvnDelete(this ICakeContext context, FilePath file)` if user writes `SvnDelete("foo")`. Should we rename to `SvnDeleteDirectory`?

Reply:
@mholo65 @christianbumann Sorry, missed that. Definitely should have unique names as you have suggested.